docs/_static/vcr/v2.yaml

```diff
@@ -423,8 +427,6 @@ components:
       $ref: '../common/ssi_types.yaml#/components/schemas/VerifiablePresentation'
     Revocation:
       $ref: '../common/ssi_types.yaml#/components/schemas/Revocation'
-    DID:
-      $ref: '../common/ssi_types.yaml#/components/schemas/DID'
```

Comment:
Just checked, the OpenAPI Generator for Java specified it as `String`, I guess since it ultimately resolves to a scalar string type. More broadly would be the question whether we want to have a DID type in our APIs, but it's already a mix and match, so it's fine by me.